controllers/v1/observationSubmissionsController.js

```diff
@@ -113,18 +117,21 @@ module.exports = class ObservationSubmissions extends Abstract {
 
         let entityDocument = { metaInformation: {} };
         if (validateEntities == 'ON') {
-          let entityQueryObject = {
+
+          let filterData = {
             _id: req.query.entityId,
             entityType: observationDocument.entityType,
-          };
-          entityDocument = await database.models.entities
-            .findOne(entityQueryObject, {
-              metaInformation: 1,
-              entityTypeId: 1,
-              entityType: 1,
-              registryDetails: 1,
-            })
-            .lean();
+           };
+ 
+           let entitiesDocument = await entityManagementService.entityDocuments(
```

Comment:
@borkarsaish65 this call should happen from helper logic. avoid such calls from controller , please

Reply:
done